### PR TITLE
Fix memory leak

### PR DESCRIPTION
**Pull request by Toktik**
**What kind of change does this PR introduce?** (check one with "x")
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
There was memory leak in the resizeable directive

**What is the new behavior?**
No memory link in resizeable directive

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/directives/resizeable.directive.ts [hunk 1]
```diff
@@ -29,16 +30,17 @@ export class ResizeableDirective implements OnDestroy, AfterViewInit {
...
     } else {
-      renderer2.addClass(node, 'resize-handle--not-resizable');
+      renderer2.addClass(this.createdNode, 'resize-handle--not-resizable');
     }
-    renderer2.appendChild(this.element, node);
+    renderer2.appendChild(this.element, this.createdNode);
   }
 
   ngOnDestroy(): void {
+    this.renderer.destroyNode(this.createdNode);
```
Comment (hugoj-goncalves) [inline src/directives/resizeable.directive.ts]: *destroyNode* is a debug only function, you should be using *removeChild* instead I guess.

**Comment by Nikcharn** [comment]
@Toktik, @hugoj-goncalves what with this pull-request? it is fix memory leaks for me

**Comment by hugoj-goncalves** [comment]
@Nikcharn well looks fine to me. My only concern with the actual code is that it will use different codes in development/production. 
*destroyNode* does not exist in production build.

Anyways, it was working fine last time I tested.

**Comment by Nikcharn** [comment]
@hugoj-goncalves you are right. We can't use 'destroyNode' in production build. If @Toktik will not be able or will not have time to correct this remark, I will create pull request with the necessary edits.

**Comment by macjohnny** [comment]
here is an alternative PR to fix this problem: https://github.com/swimlane/ngx-datatable/pull/1792

**Comment by macjohnny** [comment]
@marjan-georgiev this PR can be closed, as it was fixed in https://github.com/swimlane/ngx-datatable/pull/1792

**Pull request closed without merging**